app/models/miq_server/worker_management/monitor/validation.rb

```diff
@@ -5,7 +5,7 @@ def validate_worker(w)
     return true if MiqEnvironment::Command.is_podified?
```

Comment:
is the short term goal to have all these conditional statements in management/monitor

Reply:
So my three step plan (sound like I'm running for office) is:
1. Convert `MiqServer::WorkerManagement` from a bit mixin to a class
2. Move the current implementation into a `MiqServer::WorkerManagement::Process` and figure out what the "API" looks like for `MiqServer::WorkerManagement::Base`
3. Extract everything in a conditional to `Systemd < Base` and `Kubernetes < Base`

Reply:
Okay I've take care of most of the conditionals by moving into Kubernetes, Systemd, and Process subclasses